site/docs/config.md

```diff
@@ -198,7 +198,7 @@ vg.embed('#horizontal_line', {
 
 | Property      | Type          | Description    |
 | :------------ |:-------------:| :------------- |
-| barWidth      | Number        | The width of the bars. <span class="note-line">__Default value:__  `bandWidth-1` for the bar's dimension is an ordinal scale. (This provides 1 pixel offset between bars.) and `2` for if bar's dimension has linear scale.  </span>  |
+| barWidth      | Number        | The width of the bars. <span class="note-line">__Default value:__  `bandSize-1` for the bar's dimension is an ordinal scale. (This provides 1 pixel offset between bars.) and `2` for if bar's dimension has linear scale.  </span>  |
```

Comment:
`thickness` is worth considering, but it is not consistent with `tickThickness`  (thickness of the bar is orthogonal to thickness of thick)

barWidth can be confusing because it's the height of the bar